contrib/PyCuAmpcor/src/cuEstimateStats.cu

```diff
@@ -46,7 +46,7 @@ void cuEstimateSnr(cuArrays<float> *corrSum, cuArrays<int> *corrValidCount, cuAr
 }
 
 // cuda kernel for cuEstimateVariance
-__global__ void cudaKernel_estimateVar(const float* corrBatchRaw, const int NX, const int NY,
+__global__ void cudaKernel_estimateVar(const int winSize, const float* corrBatchRaw, const int NX, const int NY,
```

Comment:
I prefer to move winSize to the end, maybe call it templateSize.

Reply:
Have moved it to be the last input argument.